<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>

 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>520</width><height>600</height></rect>
  </property>
  <property name="windowTitle">
   <string>Register User</string>
  </property>

  <widget class="QWidget" name="centralwidget">



   <layout class="QVBoxLayout" name="mainLayout">

    <!-- TITLE -->
    <item>
      <widget class="QLabel" name="titleLabel">
        <property name="text"><string>Register New User</string></property>
        <property name="alignment"><set>Qt::AlignCenter</set></property>
      </widget>
    </item>

    <!-- FIELDS -->
    <item><widget class="QLineEdit" name="firstNameInput"><property name="placeholderText"><string>First Name</string></property></widget></item>
    <item><widget class="QLineEdit" name="lastNameInput"><property name="placeholderText"><string>Last Name</string></property></widget></item>
    <item><widget class="QLineEdit" name="emailInput"><property name="placeholderText"><string>Email</string></property></widget></item>
    <item><widget class="QLineEdit" name="passwordInput"><property name="placeholderText"><string>Password</string></property><property name="echoMode"><enum>QLineEdit::Password</enum></property></widget></item>
    <item><widget class="QLineEdit" name="cnicInput"><property name="placeholderText"><string>CNIC (42101-xxxxxxx-x)</string></property></widget></item>
    <item><widget class="QLineEdit" name="streetInput"><property name="placeholderText"><string>Street Address</string></property></widget></item>
    <item><widget class="QLineEdit" name="cityInput"><property name="placeholderText"><string>City</string></property></widget></item>

    <!-- PHONE -->
    <item><widget class="QLineEdit" name="phoneInput"><property name="placeholderText"><string>Phone Number</string></property></widget></item>

    <item><widget class="QPushButton" name="btnAddPhone"><property name="text"><string>Add Phone</string></property><property name="minimumHeight"><number>35</number></property></widget></item>

    <item><widget class="QListWidget" name="phoneList"/></item>

    <!-- STATUS -->
    <item><widget class="QLabel" name="statusLabel"><property name="text"><string></string></property></widget></item>

    <!-- SUBMIT -->
    <item><widget class="QPushButton" name="btnSubmit"><property name="text"><string>Submit Registration</string></property><property name="minimumHeight"><number>45</number></property></widget></item>

    <!-- BACK -->
    <item><widget class="QPushButton" name="btnBack"><property name="text"><string>Back</string></property><property name="minimumHeight"><number>45</number></property></widget></item>

   </layout>

  </widget>
 </widget>

 <resources/>
 <connections/>
</ui>
